code/modules/surgery/tools.dm

```diff
@@ -271,7 +274,7 @@
 
 /obj/item/surgical_drapes
 	name = "surgical drapes"
-	desc = "Nanotrasen brand surgical drapes provide optimal safety and infection control."
+	desc = "Apearature brand surgical drapes providing privacy and infection control."
```

Comment:
Typo or some kind of reference I'm missing? Did you mean "Aperture"?

Reply:
It's legally distinct Aperature, since they were a shower curtain company in Portal lore.